src/main/java/org/takes/rq/CapInputStream.java

```diff
@@ -34,6 +34,10 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.16
+ * @todo #254:30min CapInputStream should delegate all standard InputStream
```

Comment:
@exper0 why do you need this puzzle if you are no longer using `CapInputStream` in `RqMultipart`?